src/main/groovy/de/dkfz/config/ResourceSetSize.java

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
It is linked but not the same. ResourceSet defines a whole set of resources (hdd, mem, cores, nodes). The size object is a valid range of different size types. E.g. t or xs for exome and test data sets.